.github/workflows/integration.yml

```diff
@@ -0,0 +1,34 @@
+name: Integration
+on:
+  push:
+    branches:
+      - main
+      - stable-*
+  pull_request:
+
+jobs:
+  tests:
+    name: Integration Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: '1.3.6'
+          terraform_wrapper: false
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: |
+          pip install ansible-core
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '^1.18'
```

Comment:
Should this also be upgraded to 1.21 or are we intentionally supporting older go versions here?

Reply:
I would say upgrade. Go version doesn't matter in quite the same way that python version does since the user is just getting a prebuilt binary. They don't need go installed locally to run it.